packages/editor-ui/src/Interface.ts

```diff
@@ -420,13 +420,19 @@ export type IPushData =
 	| PushDataConsoleMessage
 	| PushDataReloadNodeType
 	| PushDataRemoveNodeType
-	| PushDataTestWebhook;
+	| PushDataTestWebhook
+	| PushDataNodeDescriptionUpdated;
 
 type PushDataExecutionFinished = {
 	data: IPushDataExecutionFinished;
 	type: 'executionFinished';
 };
 
+type PushDataNodeDescriptionUpdated = {
```

Comment:
Out of scope but if you want to tackle it, `PushData` should be a namespace instead of prefixed in all push types.